src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -10,7 +10,7 @@
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <TieredCompilation>false</TieredCompilation>
     <NoWarn>NU5104</NoWarn>
-    <IdentityDependencyVersion>6.31.0</IdentityDependencyVersion>
```

Comment:
cc @fabiocav @brettsam @mathewc 

This breaks dependency test with the following output. Please confirm we are good to update deps.json
```
Changed:
        - Microsoft.IdentityModel.Abstractions.dll: 6.31.0.0/6.31.0.40607 -> 6.32.0.0/6.32.0.40712
        - Microsoft.IdentityModel.JsonWebTokens.dll: 6.31.0.0/6.31.0.40607 -> 6.32.0.0/6.32.0.40712
        - Microsoft.IdentityModel.Logging.dll: 6.31.0.0/6.31.0.40607 -> 6.32.0.0/6.32.0.40712
        - Microsoft.IdentityModel.Protocols.dll: 6.10.0.0/6.10.0.20330 -> 6.32.0.0/6.32.0.40712
        - Microsoft.IdentityModel.Protocols.OpenIdConnect.dll: 6.10.0.0/6.10.0.20330 -> 6.32.0.0/6.32.0.40712
        - Microsoft.IdentityModel.Tokens.dll: 6.31.0.0/6.31.0.40607 -> 6.32.0.0/6.32.0.40712
        - System.IdentityModel.Tokens.Jwt.dll: 6.31.0.0/6.31.0.40607 -> 6.32.0.0/6.32.0.40712
    
Removed:
    
Added:
        - Microsoft.IdentityModel.Validators.dll: 6.32.0.0/6.32.0.40712
 ```

Reply:
hey @karshinlin -- can you try rebasing on dev? @fabiocav should have updated these and taken care of the test already in https://github.com/Azure/azure-functions-host/pull/9409

Reply:
@brettsam latest failure output. Can I add this new dependency?

      Changed:

  Removed:

  Added:
    - Microsoft.IdentityModel.Validators.dll: 6.32.0.0/6.32.0.40712

Reply:
You'll need to add this to the "runtimeassemblies.json" file and mark it as "private". There should be other examples in there of private assemblies (should just be a few lines of json). Just make sure to put it in alphabetical order with other assemblies.